runtime/manta/Cargo.toml

```diff
@@ -129,11 +128,9 @@ std = [
 	'frame-system/std',
 	'frame-system-rpc-runtime-api/std',
 	'frame-try-runtime/std',
-	'pallet-assets/std',
```

Comment:
Why don't we remove these as dependencies altogether.
`pallet_assets` and `pallet_proxy`